docs/contribute/testing.adoc

```diff
@@ -287,3 +299,33 @@ $ less Testing/Temporary/LastTest.log
 Results from testing will be printed on the stdout and detailed log file with
 your test results can be found in the `Testing/Temporary/LastTest.log` file.
 
+== Running the MITRE tests
+
+The MITRE tests (in `tests/mitre`) are an example of how not to write tests.
```

Comment:
I think we shouldn't put this evaluation here, because MITRE is an external contributor and we are happy that they provided us the tests.

Reply:
Ah sorry. Fair point, I'll update.